diff -Nurp firewalld-0.6.3_orig/src/firewall/core/ipXtables.py firewalld-0.6.3/src/firewall/core/ipXtables.py
--- firewalld-0.6.3_orig/src/firewall/core/ipXtables.py	2019-05-03 20:24:14.324627074 +0200
+++ firewalld-0.6.3/src/firewall/core/ipXtables.py	2019-05-03 20:30:11.406784867 +0200
@@ -549,6 +549,8 @@ class ip4tables(object):
     def build_flush_rules(self):
         rules = []
         for table in BUILT_IN_CHAINS.keys():
+            if not self.get_available_tables(table):
+                continue
             # Flush firewall rules: -F
             # Delete firewall chains: -X
             # Set counter to zero: -Z
@@ -559,6 +561,8 @@ class ip4tables(object):
     def build_set_policy_rules(self, policy):
         rules = []
         for table in BUILT_IN_CHAINS.keys():
+            if not self.get_available_tables(table):
+                continue
             if table == "nat":
                 continue
             for chain in BUILT_IN_CHAINS[table]:
